roles/schema/defaults/main.yml

```diff
@@ -0,0 +1,35 @@
+---
+# connectivity variables
+sr_port: 8081  
+proxy_env: {}
+ansible_fqdn: ""
+topic_protected: ^_confluent.*|^_connect.*|^_ksql.*|^_schemas|^confluent.*
+topics_key_t0: []
+topics_value_t0: []
+topics_key: []
+topics_value: []
+key_flat: []
+key_flat_add: []
+value_flat: []
+value_flat_add: []
+Combined_flat: []
```

Comment:
Does not seem to be used anywhere. Can you please check if all the other variables are needed?

Reply:
All the above variables are used within schemas_manage.yml.